packages/web-components/fast-components/src/slider/slider.styles.ts

```diff
@@ -61,38 +62,45 @@ export const sliderStyles: (
         height: 100%;
         grid-template-columns: calc(var(--thumb-size) * 1px) 1fr;
     }
-
-    :host(:${focusVisible}) .thumb-cursor {
-        box-shadow: 0 0 0 2px ${fillColor}, 0 0 0 4px ${focusStrokeOuter};
-    }
-
     .thumb-container {
         position: absolute;
         height: calc(var(--thumb-size) * 1px);
         width: calc(var(--thumb-size) * 1px);
         transition: all 0.2s ease;
-        color: ${neutralForegroundRest};
-        fill: currentcolor;
     }
     .thumb-cursor {
+        display: flex;
+        position: relative;
         border: none;
         width: calc(var(--thumb-size) * 1px);
         height: calc(var(--thumb-size) * 1px);
-        background: ${neutralForegroundRest};
-        border-radius: calc(${controlCornerRadius} * 1px);
+        background: ${fillColor};
+        border: calc(${focusStrokeWidth} * 1px) solid ${strokeControlStrongRest};
+        border-radius: 50%;
+        box-sizing: border-box;
     }
-    .thumb-cursor:hover {
-        background: ${neutralForegroundRest};
-        border-color: ${neutralStrokeHover};
+    :host(:not([disabled])) .thumb-cursor:hover,
+    :host(:not([disabled])) .thumb-cursor:active {
+        background: ${neutralForegroundHover};
+        border-color: ${strokeControlStrongActive};
     }
-    .thumb-cursor:active {
-        background: ${neutralForegroundRest};
+    :host(:${focusVisible}) .thumb-cursor {
+        background: ${neutralForegroundHover};
+        border-color: ${focusStrokeOuter};
```

Comment:
Let's follow the same pattern as checkbox and radio here as well. So we keep the normal border and offset the outline with a gap from the non-focused thumb.

Reply:
updated